cumulusci/core/config/project_config.py

```diff
@@ -727,11 +733,53 @@ def process_github_dependency(  # noqa: C901
 
         return repo_dependencies
 
-    def _find_latest_release(self, repo, include_beta=None):
-        try:
-            if include_beta:
-                return next(repo.releases())
+    def get_task(self, name):
```

Comment:
Some doc strings for `get_task` and other methods would be good.

Reply:
Added!